tests/shell/shell_recorder/shell_recorder.sh

```diff
@@ -295,7 +295,7 @@ EVAL=0
 
```

Comment:
It already says the solution (`kdb import < $TMPFILE`), but it is only a performance improvement.